hashes/src/sha256t.rs

```diff
@@ -27,7 +27,13 @@ impl<T: Tag> schemars::JsonSchema for Hash<T> {
     fn schema_name() -> String { "Hash".to_owned() }
 
     fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
-        crate::util::json_hex_string::len_32(gen)
```

Comment:
I think you could've just called `sha256::Hash::json_schema(gen)`.

Reply:
Ha! Nice one.